apps/src/applab/designMode.js

```diff
@@ -105,6 +105,26 @@ designMode.resetElementTray = function (allowEditing) {
 };
 
 /**
+ * If the filename is relative (contains no slashes), then prepend
+ * the path to the assets directory for this project to the filename.
+ * @param {string} filename
+ * @returns {string}
+ */
+designMode.maybeAddAssetPathPrefix = function (filename) {
+  filename = filename || '';
+  if (filename.indexOf('/') !== -1) {
```

Comment:
that wouldn't work for image urls on other domains. The asset uploader already replaces '/' with ':', at least on chrome, so I wouldn't expect any of our filenames to contain '/'. If we want to support nested files in the future, we could check for prefixes ["https://", "http://", "/"] and treat everything else as relative paths, although it's unclear to me if maybe this would rule out any other valid image paths. Thoughts?